src/middlewares/CheckRoleStatus.js

```diff
@@ -0,0 +1,43 @@
+import models from '../models';
+import response from '../utils/Response';
+
+const { serverResponse } = response;
+
+const { Users, roles } = models;
+
+/**
+ * @class CheckStatus
+ */
+class RoleStatus {
+  /**
+ *
+ *
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Object} next
+ * @returns {Integer} userRole
+ * @memberof RoleStatus
+ */ // eslint-disable-next-line
```

Comment:
is there a reason we are disabling eslint here?

Reply:
The line length is longer than 110 characters as per eslint check. I cannot seem to break the code into a second line so I had to disable the rule